document/src/vespa/document/fieldvalue/structuredfieldvalue.cpp

```diff
@@ -18,15 +18,13 @@ using namespace fieldvalue;
 IMPLEMENT_IDENTIFIABLE_ABSTRACT(StructuredFieldValue, FieldValue);
 
 StructuredFieldValue::Iterator::Iterator()
-    : _owner(0),
-      _iterator(),
+    : _iterator(),
       _field(0)
 {
 }
 
-StructuredFieldValue::Iterator::Iterator(const StructuredFieldValue& owner, const Field* first)
-    : _owner(&const_cast<StructuredFieldValue&>(owner)),
-      _iterator(owner.getIterator(first).release()),
+StructuredFieldValue::Iterator::Iterator([[maybe_unused]] const StructuredFieldValue& owner, const Field* first)
```

Comment:
`owner` is used as part of `_iterator` construction below, remove `[[maybe_unused]]`?